gateway/src/main/java/io/zeebe/gateway/impl/broker/RequestRetriesExhaustedException.java

```diff
@@ -9,7 +9,9 @@
 
 import io.zeebe.gateway.cmd.ClientException;
 
-class RequestRetriesExhaustedException extends ClientException {
+public class RequestRetriesExhaustedException extends ClientException {
+
+  private static final long serialVersionUID = 1493435667474886479L;
```

Comment:
Do you think it is necessary?

Reply:
No, sorry, my IDE automatically adds it :see_no_evil: